README.md

```diff
@@ -34,6 +34,7 @@ Below is a list of protections detected by BurnOutSharp. The two columns explain
 | BD+ | False | True | |
 | Bitpool | False | True | |
 | ByteShield | False | True | Unconfirmed¹ |
+| C-Dilla License Management Solution / CD-Secure / CD-Compress | True | True | |
```

Comment:
I hate this, but I understand why it has to be documented like this.

Reply:
I don't like it any better :(